src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -161,7 +154,7 @@ const DiscoverySearch = ({
   }, []);
 
   return (
-    <View style={styles.animatedViewContainer}>
+    <View style={styles.animatedViewContainer(hideBackIcon)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style `animatedViewContainer` now receives a parameter `hideBackIcon`. Ensure that this parameter is passed correctly wherever this style is used.
```diff
- <View style={styles.animatedViewContainer}>
+ <View style={styles.animatedViewContainer(hideBackIcon)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->